ui-tests/src/test/java/org/openmrs/reference/BookRequestAppointmentTest.java

```diff
@@ -42,8 +42,8 @@ public void bookRequestAppointmentTest() throws Exception {
         ClinicianFacingPatientDashboardPage patientDashboardPage = activeVisitsPage.goToPatientDashboardOfLastActiveVisit();
         RequestAppointmentPage requestAppointmentPage = patientDashboardPage.clickOnRequest();
         requestAppointmentPage.enterAppointmentType("Oncology");
-        requestAppointmentPage.enterValue("0");
-        requestAppointmentPage.selectUnits("Day(s)");
+        requestAppointmentPage.enterMinimumValue("0");
+        requestAppointmentPage.selectMinimumUnits("Day(s)");
```

Comment:
Refactored `function` names [h](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/9c172b398f03b79b816894c176df11643432a6c2/ui-tests/src/test/java/org/openmrs/reference/page/RequestAppointmentPage.java#L31)er[e](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/9c172b398f03b79b816894c176df11643432a6c2/ui-tests/src/test/java/org/openmrs/reference/page/RequestAppointmentPage.java#L35) for better java naming convention since `selectMaximumValue()` and `selectMaximumUnits()` were introduced

Reply:
Next time, use a separate pull request for refactoring.